hieradata/clients/azure.ci.jenkins.io.yaml

```
@@ -30,6 +30,7 @@ datadog_agent::apm_enabled: true
 datadog_agent::apm_non_local_traffic: true # Allow jenkins container to contact the agent for traces and logs
 datadog_agent::agent_extra_options:
   bind_host: "0.0.0.0" # All hosts interfaces to allow container access
+datadog_agent::container_collect_all: true # Don't collect Jenkins container logs (the only container running) as we're already collecting these logs via the Datadog plugin
```

```suggestion
datadog_agent::container_collect_all: false # Don't collect Jenkins container logs (the only container running) as we're already collecting these logs via the Datadog plugin
```

Ref.
- https://forge.puppet.com/modules/datadog/datadog_agent/readme 

> container_collect_all | A boolean to enable logs collection for all containers.

- https://github.com/DataDog/puppet-datadog-agent/blob/57ffbd188b0b32678328e3e4ac743b44b8c6e3c7/manifests/init.pp#L662

---

🤦